File: 프로그래머스/lv0/120907. OX퀴즈/OX퀴즈.py
def solution(quiz):
    answer = []
    for q in quiz:
        list = q.split()

        if list[1] == '+':
            if int(list[0]) + int(list[2]) == int(list[4]):
                answer.append('O')
            else:
                answer.append('X')
        elif list[1] == '-':
            if int(list[0]) - int(list[2]) == int(list[4]):
                answer.append('O')
            else:
                answer.append('X')
    return answer

File: 프로그래머스/lv0/120907. OX퀴즈/OX퀴즈.java
class Solution {
    public String[] solution(String[] quiz) {
        String[] answer = new String[quiz.length];
        String[] a = new String[quiz.length];
        
        for (int i = 0; i < quiz.length; i++) {
            a = quiz[i].split(" ");
            if (a[1].equals("+")) {
                if (Integer.parseInt(a[0]) + Integer.parseInt(a[2]) == Integer.parseInt(a[4])) {
                    answer[i] = "O";
                } else {
                    answer[i] = "X";
                }
            }
            if (a[1].equals("-")) {
                if (Integer.parseInt(a[0]) - Integer.parseInt(a[2]) == Integer.parseInt(a[4])) {
                    answer[i] = "O";
                } else {
                    answer[i] = "X";
                }
            }
        }
        return answer;
    }
}